ideas/150-gas-abstraction.md

```diff
@@ -1,3 +1,21 @@
+---
+id: 150-gas-abstraction
+title: Gas Abstraction
+status: Draft
+created: 2018-02-01
+category: core
+lead-contributor: Ricardo Guilherme Schmidt
+contributors:
+    - Ricardo Guilherme Schmidt
```

Comment:
Would you mind syncing these to contributors.yaml file / add GH names if it makes sense? E.g. Ricardo should be @3esmit, Richard @richard-ramos @iurimatias, Alex (from Mist IIRC) @alexvandesande.

Reply:
Sure - what about the handles for Hester Bruikman, Denis Sharypin, and Rachel Hamlin?